Misc/NEWS.d/next/Library/2024-03-09-12-55-44.gh-issue-108901.s9VClL.rst

```diff
@@ -0,0 +1,3 @@
+Add :meth:`inspect.Signature.from_frame` to get a singature from a frame
+object, it will be used as a new alternative for older
+:func:`inspect.getargvalues` and :func:`inspect.formatargvalues`.
```

Comment:
"it will be used as a new alternative for older ..." I don't think that it's useful to mention this in the Changelog entry.